### PR TITLE
Use yarn dlx instead of npx

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

**Comment by dunglas** [comment]
Do not merge until the official release of Yarn 2.

File: client-generator/nextjs.md [hunk 1]
```diff
@@ -10,8 +10,6 @@ The Next.js Client Generator generates components for Server Side Rendered appli
 
 Create a [Next.js application with express server](https://github.com/zeit/next.js/tree/canary/examples/custom-server-express). The easiest way is to execute:  
 
-    $ npx create-next-app your-app-name
-    # or
     $ yarn create next-app your-app-name
```
Comment (teohhanhui) [inline client-generator/nextjs.md]: There is no longer a `yarn create` command in yarn v2: https://next.yarnpkg.com/cli/create

As it was [a shortcut](https://classic.yarnpkg.com/en/docs/cli/create/), we should now do:

    $ yarn dlx create-* ...

Comment (teohhanhui) [inline client-generator/nextjs.md]: There is no longer a `yarn create` command in yarn v2.

As it was [a shortcut](https://classic.yarnpkg.com/en/docs/cli/create/), we should now do:

    $ yarn dlx create-* ...

Comment (dunglas) [reply client-generator/nextjs.md]: But Yarn 2 cannot be installed globally (you need Yarn 1 to install it) so it doesn't work.

**Pull request closed without merging**